src/Common/Session/SessionTracker.php

```diff
@@ -39,12 +41,16 @@ public static function isSessionExpired(): bool
             error_log("OpenEMR Error: session_database_uuid session variable is missing");
             return true;
         }
-        $last_updated = sqlQueryNoLog("SELECT `last_updated` FROM `session_tracker` WHERE `uuid` = ?", $_SESSION['session_database_uuid'])['last_updated'];
-        if (empty($last_updated)) {
+        $sessionTracker = sqlQueryNoLog("SELECT `last_updated`, NOW() as `current_time` FROM `session_tracker` WHERE `uuid` = ?", $_SESSION['session_database_uuid']);
```

Comment:
@stephenwaite , ended up being easier to just base all the sessionExpire stuff on the mysql/mariadb server. So, now should always be consistent since time is coming from one place.